pull_request_template.md

```diff
@@ -0,0 +1,91 @@
+<!--Thank you for contributing to AllenSDK, your work and time will help to
+advance open science! For full contribution guidelines check out our
+guide on GitHub here, https://github.com/AllenInstitute/AllenSDK/blob/master/CONTRIBUTING.md-->
+
+# Overview:
+<!-- Give a brief overview of the issue you are solving. Succinctly
+explain the GitHub issue you are addressing and the underlying problem
+of the ticket. The commit header and body should also include this
+message, for good commit messages see the full contribution guidelines.
+example: 
+Science team is not able to load max or avg projections for experiment
+session #x. A image cannot be created because input pixel
+resolution is (0,0). It was found through investigation that the
+experiment database query was returning a 0 pixel resolution for this
+experiment.-->
+
+# Addresses:
+<!-- Add a link to the issue on Github board
+example: 
+Addresses issue [#1234](git_hub_ticket_url)-->
+
+# Type of Fix:
+<!--Chose One-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing
+      functionality to not work as expected)
+- [ ] Documentation Change
+
+# Solution:
+<!-- Outline your solution to the previously described issue and
+underlying cause. This section should include a brief description of
+your proposed solution and how it addresses the cause of the ticket
+example:
+Solution to this problem is to update the value of the pixel resolution
+to a default x if pixel resolution is database pixel resolution =0. This
+will address the underlying problem by providing a fallback value if
+the data is not available. A downfall is if default resolution is disparate
+from actual resolution that wasn't saved, images might appear very distorted.
+An alternative solution is to update the database to cover the missing 
+experiment resolutions.-->
+
+# Adds:
```

Comment:
might want to rename this section since changes are not necessarily additive

Reply:
Updated to changes, I agree it might be misleading